test/functional/feature_poolswap_mechanism.py

```diff
@@ -319,7 +319,7 @@ def run_test(self):
         assert(self.nodes[0].getblockcount() == 120) # eunos
 
         self.LP_DAILY_DFI_REWARD = self.nodes[0].getgov("LP_DAILY_DFI_REWARD")['LP_DAILY_DFI_REWARD']
-        assert_equal(self.LP_DAILY_DFI_REWARD, Decimal('103.08268000'))
+        assert_equal(self.LP_DAILY_DFI_REWARD, Decimal('14843.90592000')) # 144 blocks a day times 103.08268000
```

Comment:
Why is it 144? Functional test has a much longer block time?

Reply:
Regtest is still on Bitcoin block times.

https://github.com/DeFiCh/ain/blob/118b8be697cc880de2a23b5a47b6031d3b49ef2b/src/chainparams.cpp#L697